src/lib/OTA/OTA.cpp

```diff
@@ -49,7 +49,7 @@ void ICACHE_RAM_ATTR GenerateChannelDataHybridSwitch8(volatile uint8_t* Buffer,
   uint8_t bitclearedSwitchIndex = nextSwitchIndex - 1;
   // currentSwitches[] is 0-15 for index 1, 0-2 for index 2-7
   // Rely on currentSwitches to *only* have values in that rang
-  uint8_t value = crsf->currentSwitches[nextSwitchIndex];
+  uint8_t value = crsf->currentSwitches[nextSwitchIndex] & 0b11;
```

Comment:
Should this be a 3 bit mask? The comments seem to have got a bit out of whack, maybe caused confusion.

Reply:
Yup! Actually it should be 4 bits sometimes and 3 bits other times. Fixed both the comment and the code